docs/mobile/development.md

```diff
@@ -0,0 +1,104 @@
+# Development Process
+
+## Set Up
+
+Each mobile courses is a GitHub repo under the DataCamp GitHub organization. You
+will be given full write access to the repository for your course. First clone
+the repository:
+
+```sh
+git clone git@github.com:datacamp/mobile-courses-example-course.git
+```
+
+The suite of development tools available for authoring mobile content on
+DataCamp is provided as an NPM package. You can download and install it using
+NPM. [Install NPM](https://www.npmjs.com/get-npm), then:
+
+```sh
+npm install -g @datacamp/mobile-teach
+```
+
+This will create the `mobile-teach` executable, which you will use to lint and
+preview your content.
+
+## Write
+
+You are free to use whatever editing tools you like on your local repository, eg
+Sublime, Vim, Emacs, etc. You should author content on a feature branch, *not*
+on the `master` branch:
```

Comment:
This is important - highlight it.

Related engineering issue: Is the master branch protected by default?

Reply:
@entropitor, no I don't think it is.

Reply:
Nope, at the moment it is not, but it should not be too hard to enable this, if you want :)